sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -814,7 +814,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     withTempView("tab1") {
       sql(
         """
-          |CREATE TEMPORARY TABLE tab1
```

Comment:
For this one, I think we should keep the test coverage because we still support this legacy syntax.
Could you revert this kind of changes?

Reply:
Ok, done. I keep the old test cases for test coverage, and add new test cases for temp view